rem/DB.swift

```diff
@@ -59,6 +59,9 @@ class DatabaseManager {
             db = try! Connection("db.sqlite3")
         }
         
+        // 2 second busy timeout
+        db.busyTimeout = 2000
```

Comment:
My initial reaction to this is- we're not doing writes from outside, just reads right? We should just support concurrent reads. Maybe something like:

```swift
try db.run("PRAGMA journal_mode = WAL")
// Maybe?
try db.run("PRAGMA synchronous = NORMAL")
```